src/Core/Checkout/Cart/CartException.php

```diff
@@ -634,4 +635,13 @@ public static function cartLocked(string $token): self
             ['token' => $token]
         );
     }
+
+    public static function serializedCartTooLarge(): self
+    {
+        return new self(
+            Response::HTTP_REQUEST_ENTITY_TOO_LARGE,
```

Comment:
I don't think this is the correct status code. 413 should be returned, if the payload of the request is too large, not the result of some internal process - we should return 400 or maybe 422.